.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+node_js:
+  - "6.12.0"
+  - "node"
+  - "lts/*"
+  - "8"
+  - "7"
+  - "6"
+  - "5"
+  - "4"
+
+cache: yarn
+
+before_script:
+  - yarn global add bower
+  - bower -v
+  - bower i
+  - yarn global add phantomjs
+  - phantomjs --version
+  - yarn global add mocha
+  - mocha --version
+  - yarn global add mocha-phantomjs
+  - mocha-phantomjs --version
+
+script:
+  - yarn run test
```

Comment:
this should just be `yarn run build`, then the `test` script doesn't have to be modified. is that correct?

Reply:
and don't forget to add Mocha to `devDependencies` in package.json

Reply:
I'm not sure what you mean by this:

> this should just be `yarn run build`, then the `test` script doesn't have to be modified. is that correct?

Do you mean that this is travis's default behaviour so we could delete the `script` part? Since there are other commands after this one, I think this is necessary.

> and don't forget to add Mocha to `devDependencies` in package.json

Reply:
I'm suggesting the following revisions

**travis.yml**

```diff
 script:
-  - yarn run test
+  - yarn run build
   - mocha-phantomjs ./test/test.html
   - yarn run spec
```

**package.json**

```diff
 "scripts": {
-  "test": "npm run build",
+  "test": "npm run build && open ./test/test.html",
   "spec": "mocha -u tdd --reporter spec",
   "build": "npm run minify",
   "minify": "uglifyjs jquery.serialize-object.js -m -c --comments > dist/jquery.serialize-object.min.js"
 },

...

 "devDependencies": {
+  "mocha": "^5.0.1",
   "uglify-js": "^2.4.16"
 }
```
Does that make sense?

Reply:
In hindsight, the `build` and `minify` scripts cross concerns. It's a separate issue but should be patched as

```diff
-  "build": "npm run minify",
+  "build": "npm run minify > dist/jquery.serialize-object.min.js",
-  "minify": "uglifyjs jquery.serialize-object.js -m -c --comments > dist/jquery.serialize-object.min.js"
+  "minify": "uglifyjs jquery.serialize-object.js -m -c --comments"
```

If you want to sneak that in this PR, I'm cool with that